frontend/utils/backend.ts

```
@@ -0,0 +1,14 @@
+/**
+ * Centralized backend URL configuration
+ */
+
+function normalizeBaseUrl(url: string): string {
+  return url.replace(/\/+$/, "");
```

ESLint

[error] Use the 'u' flag.

(require-unicode-regexp)